src/platform/ESP32/ConfigurationManagerImpl.cpp

```diff
@@ -122,6 +139,10 @@ CHIP_ERROR ConfigurationManagerImpl::Init()
     SuccessOrExit(MapConfigError(esp_err));
     esp_err = nvs_flash_init_partition(CHIP_DEVICE_CONFIG_CHIP_KVS_NAMESPACE_PARTITION);
     SuccessOrExit(MapConfigError(esp_err));
+    esp_err = nvs_flash_init_partition(CHIP_DEVICE_CONFIG_CHIP_DEVICE_ATTESTATION_PARTITION);
+    SuccessOrExit(MapConfigError(esp_err));
+    esp_err = nvs_flash_init_partition(CHIP_DEVICE_CONFIG_CHIP_CERTIFICATION_DECLARATION_PARTITION);
```

Comment:
CHIP_DEVICE_CONFIG_CHIP_DAC_PRIV_KEY_PARTITION does not need to be inited?  Worth adding a comment saying that and explaining why, so someone doesn't add it here.  Similar for the secure init bits above.

Reply:
The CHIP_DEVICE_CONFIG_CHIP_DAC_PRIV_KEY_PARTITION is not a nvs sub-type partition in this PR. I use `esp_partition_read` to get the dac private key.